### ISSUE TITLE
[News] Massive update incoming, your code may break!

### ISSUE THREAD

**Issue by realcoloride** [realcoloride/node_characterai#10]
Hello!
I have worked on a new update today that brings tons of changes to the API wrapper!

Many classes were added.
Thanks to all the people who gave some tips and feedback.

#### `Reply`
Returned by `chat.sendAndAwaitResponse()`
Contains the information of a reply.

To manage or interact with the message, use the `await reply.getMessage()` function.
When printed using `console.log`, it will return the text property (for testing purposes), this is an experimental feature, but will be disabled by feedback if needed.

⚠️ **So make sure that you use `reply.text` in your code!**
This will allow you to interact and get more information about a reply.

If you dont use `singleReply`, you will get an array of `Reply` (replies).

#### `Message`
Primary message class that allows you to interact with the message and get proper information about it.

Notable methods:
`message.delete(deletePreviousToo = false)` - Reference to issue #6, allows you to delete the message sent, and delete the previous one (can be the AI message that preceeds yours)
`message.getPreviousMessage()` - Gets you the message that preceeds this one, returns null if it doesn't exist.
 
#### `OutgoingMessage`
Class message to send a message with more information than a text message.
Used in `chat.sendAndAwaitResponse(options, singleReply)`
**👌 You will still be able to use text instead of this class.**

#### `MessageHistory`
Class that contains an array of messages when fetched with `chat.fetchHistory()`.
Contains also extra details, but use `.messages` to access those messages.

### `Changes to the Chat class`
`deleteMessagesBulk(amount = 50)` - #6 Deletes messages in mass by amount
`deleteMessage(messageId)` - #6 Deletes a message by its id
`getMessageById` - Gets a message (class) by its id to interact with

**TL:DR** - Added support for message deletion, conversation management, more parameters for messages, checks...
This update may break some stuff of code out there, so please check!

This update will be pushed soon, I am just making extra tests.

**Comment by realcoloride** [comment]
Update `1.1.0` is available for use!
Please update to the latest version and leave feedback :)

Have fun!
